### ISSUE TITLE
If I want to add dropout layers

### ISSUE THREAD

**Issue by yanxiangyi** [pierluigiferrari/ssd_keras#63]
Hi @pierluigiferrari ! Thanks for your great work!

Currently, I encountered some problems that I actually want to add some dropout layers in this net to have some experiments. 

If I just modify the keras_ssd300.py by adding some dropout layers in the code, would the h5 file still usable for me? Or do I have to train from scratch?

**Comment by pierluigiferrari** [comment]
You can just add the dropout layers to the SSD300 and load the trained weights into the resulting model without issues. The performance of the trained weights will remain unaffected of this. Of course you'll have to load the weights with `by_name=True`, but you should always do that anyway.
